.github/CODEOWNERS

```diff
@@ -918,6 +918,9 @@
 # ServiceLabel: %VideoAnalyzer
 # ServiceOwners:                                                   @giakas
 
+# ServiceLabel: %VirtualEnclaves
```

Comment:
After you have a valid label, this will need to move into the management section that starts on L955 and include `%Mgmt`.    Entries should be ordered alphabetically by label name.